apps/chargeanim/app.js

```diff
@@ -27,9 +77,23 @@ function anim() {
   });
   g.drawImage(imgbat, mx,my,{scale:b2scale, rotate:Math.sin(getTime()*2)*0.5-Math.PI/2 + b2rot});
   g.flip();
+  
+  if(settings.showTime==true){
+    g.drawString(clockStr,x,cy);
+  }
+  if(settings.showBatPercent==true){
+    g.drawString(batteryPercentStr,x,by,true);
```

Comment:
Can you put these 2 drawStrings *before* `g.flip()` please, so it's at the end? The flip is what force-pushes the graphics to the screen, so once you do that all the flicker should be gone!